### PR TITLE
wabt testing and mut globals fix

### PR DESCRIPTION
**Pull request by yurydelendik**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "spec"]
+	path = spec
+	url = https://github.com/WebAssembly/spec
```
Comment (sunfishcode) [inline .gitmodules]: Another option is to use `https://github.com/WebAssembly/testsuite`, which is a mirror of just the `test/core` directory of the `spec` repo, though I don't have a strong opinion about which you use here.

**Review by sunfishcode — APPROVED** [review]
It's not clear whether anything is testing `reloc.wasm` after this PR.

Otherwise, this looks good.

File: .gitmodules (hunk 1 above)
Comment (yurydelendik) [reply .gitmodules]: reloc.wasm was an object file with const STACKTOP and had set_global -- pre-mutable globals spec file. I changed STACKTOP to be mutable.

Comment (yurydelendik) [reply .gitmodules]: Okay, switched to WebAssembly/testsuite

**Pull request merged**